modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -3036,4 +3036,47 @@ definition {
 		DetailsView.deleteEntry(itemName = "Changed");
 	}
 
+	@description = "This is a use case for LPS-95964"
+	@priority = "5"
+	test ViewFieldSetsInTableView {
+		property test.name.skip.portal.instance = "AppBuilder#ViewFieldSetsInTableView";
```

Comment:
I forgot to mention that you can remove this line of code as well. There is no need to skip virtual instance for this test. Thanks!